content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -19,6 +19,8 @@ instead.
 
 1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
 
+1. Download the [{{< istioctl >}} command](/docs/ops/diagnostic-tools/istioctl/).
```

Comment:
Should this say "Install" instead of "Download"? The linked-to page talks about installing, and has steps beyond just downloading.

Reply:
Good point, fixed in a followup PR: #6987